app/models/json_builders/channels_json_builder.rb

```diff
@@ -19,16 +19,14 @@ class JsonBuilders::ChannelsJsonBuilder
   def initialize
     require "publishers/excluded_channels"
     @excluded_channel_ids = Publishers::ExcludedChannels.brave_publisher_id_list
-    @verified_channels = Channel.verified
     @excluded_verified_channel_ids = []
   end
 
   def build
     channels = []
 
-    [Channel.verified.site_channels, Channel.verified.youtube_channels, Channel.verified.twitch_channels, Channel.verified.twitter_channels].each do |verified_channels|
+    [Channel.verified.site_channels.includes(:site_banner).includes(publisher: :site_banners), Channel.verified.youtube_channels.includes(:site_banner).includes(publisher: :site_banners), Channel.verified.twitch_channels.includes(:site_banner).includes(publisher: :site_banners), Channel.verified.twitter_channels.includes(:site_banner).includes(publisher: :site_banners)].each do |verified_channels|
```

Comment:
This is a good optimization, may want to also test this on staging to make sure the output is the same